BookReader/BookReader.js

```diff
@@ -223,6 +225,7 @@ BookReader.prototype.setup = function(options) {
 
     this.animating = false;
     this.flipSpeed = options.flipSpeed;
+    this.flipDelay = options.flipDelay;
```

Comment:
If flipDelay is not used anywhere else in book reader, you may want to move this property into the autoPlay plugin.

Reply:
That's true. I'll move it over.